sklearn/preprocessing/_function_transformer.py

```diff
@@ -46,13 +46,18 @@ class FunctionTransformer(BaseEstimator, TransformerMixin):
         Indicate that transform should forward the y argument to the
         inner callable.
 
+    kw_args : dict, optional
+        Dictionary of additional keyword arguments to pass to func.
+
     """
     def __init__(self, func=None, validate=True,
-                 accept_sparse=False, pass_y=False):
+                 accept_sparse=False, pass_y=False,
+                 kw_args=None):
```

Comment:
why now ``kwargs``? because that could be misread as being kwargs of this function?